### ISSUE TITLE
repomd.xml may not be updated in some cases

### ISSUE THREAD

**Issue by g4s8** [artipie/rpm-adapter#26]
`Rpm.update()` may not include references to `primary`, `filelists` and `other` to `repomd.xml` file in some cases, bug other metadata files seems to be ok.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@e6cb530cd606, thread: PQ-CT2E6TK9B
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 2 error(s) in XML document: -1:-1: cvc-pattern-valid: Value 'gh:Adapter back end integration' is not facet-valid with respect to pattern '[a-z]{2}:[A-Z0-9a-z.\-#/]+' for type 'milestoneID'.;-1:-1: cvc-complex-type.2.2: Element 'successor' must have no element [children], and the value must be valid.
```

[0.51.7](https://github.com/zerocracy/farm): CID: [ce470b4e-8f94-449f-b767-4e7faa176e0c](https://www.0crat.com/footprint/CT2E6TK9B/ce470b4e-8f94-449f-b767-4e7faa176e0c), Type: "Job milestoned"

**Comment by 0crat** [comment]
The job [#26](https://github.com/artipie/rpm-adapter/issues/26) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/c7731be1-d6f2-45a0-979a-6ca41b370611); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
It is strongly discouraged to assign jobs to their creators, see [§19](http://www.zerocracy.com/policy.html#19): -15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by g4s8** [comment]
ASTO bug should be fixed in https://github.com/artipie/asto/pull/92

**Comment by g4s8** [comment]
Fixed in #27 and https://github.com/artipie/asto/issues/91
